solution/ui/regulations/eregs-vite/src/components/dropdowns/Categories.vue

```diff
@@ -45,8 +45,10 @@ onBeforeMount(() => {
     const { categories, intcategories } = $route.query;
 
     if (categories) {
+        silentReset.value = true;
         selectedId.value = `${categories}-categories`;
     } else if (intcategories) {
+        silentReset.value = true;
         selectedId.value = `${intcategories}-intcategories`;
```

Comment:
`selectedId` is being `watch`ed later in this file; changing it triggers a route update UNLESS you set `silentReset` to `true`.  When we have a category in the URL on load, we need to set the Categories dropdown to have that value selected; this was triggering a route update, which was scrubbing `page` from the route if it existed.  So here we set `silentReset` to `true` so URL params aren't scrubbed out of the route on load.